docs/server-side/v2/policy/create-template-via-api.mdx

```diff
@@ -442,14 +442,14 @@ curl --location --request POST 'https://api.100ms.live/v2/templates' \
+| --------------- | :------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------- |
+| name            | `string` | Alias for the role associated with a particular template.                                                                                                                                                                                                                                                                                                     | No       |
+| publishParams   | `object` | Publish parameters for this role. <br/><br/>Object of type [publishParams](#publishparams-object) with the collection of objects such as `allowed`, `audio`, `video`, `screen`, etc                                                                                                                                                                           | No       |
+| subscribeParams | `object` | Subscribe parameters for this role. <br/><br/>Object of type [subscribeParams](#subscribe-object) with a mix of arguments and objects like `subscribeToRoles`, `maxSubsBitRate`, & `subscribeDegradation`                                                                                                                                                     | No       |
+| permissions     | `object` | Permissions for this role. <br/><br/>Object of type [permission](#permissions-object) with collection of objects like `browserRecordings`, `rtmpDestinations`, & `hlsDestinations`.                                                                                                                                                                           | No       |
+| priority        | `int`    | Priority of the role for subscribing/publishing. <br/><br/>Priority will determine the order in which the roles will be degraded. A lower number represents a higher priority.                                                                                                                                                                                | No       |
+| maxPeerCount    | `int`    | The number of peers allowed for a particular role in a session. <br/><br/> `Default`: 0 (unlimited). <br/><br/> `Allowed values`:<br/><ul><li>**-1** : no peer will be allowed to join with that role.</li><li>**0** : unlimited.</li><li>**30** : only 30 peers are allowed to join with that role (you can set a different custom limit as well).</li></ul> | No       |
```

Comment:
> Allowed values: 

remove this line since it's obvious. also why is it in back ticks? is it a code snippet?

> -1 : no peer will be allowed to join with that role.
0 : unlimited.
30 : only 30 peers are allowed to join with that role (you can set a different custom limit as well).

why are we choosing 30? just write `n` to ensure it can take any values. if you're saying "allowed values" on top, why have only -1, 0, 30? with `n` you can remove the bracket as well, which is not required

few more things
* why is 0 not say that it is the default?
* why is -1 on the top? please put the list in reverse order; users want to know `n` and not -1
*

Reply:
made the changes, reg quality - will keep in mind going forward.